testing/sdk_metrics.py

```diff
@@ -7,12 +7,12 @@
 
 import shakedown
 
-import sdk_cmd as cmd
+import sdk_cmd
 
 log = logging.getLogger(__name__)
 
 
-def get_metrics(service_name, task_name):
+def get_metrics(service_name, package_name, task_name):
```

Comment:
Could you go with `package_name, service_name, task_name` here? Fits a little better (and also trying to standardize that via PR #1446 )

Reply:
Done